gp-cli/src/main/java/com/ibm/g11n/pipeline/tools/cli/GPCmd.java

```diff
@@ -26,6 +26,11 @@
  * @author Yoshito Umaoka
  */
 public class GPCmd {
+    // short term workaround for TLS version issue on Java 7
```

Comment:
Should this check the java version? and if > Java 7, let Java choose the default?